src/modules/facility-guarantee/facility-guarantee.controller.test.ts

```diff
@@ -0,0 +1,52 @@
+import { GetFacilityGuaranteesGenerator } from '@ukef-test/support/generator/get-facility-guarantees.generator';
+import { RandomValueGenerator } from '@ukef-test/support/generator/random-value-generator';
+import { when } from 'jest-when';
+
+import { DateStringTransformations } from '../date/date-string.transformations';
+import { FacilityGuaranteeController } from './facility-guarantee.controller';
+import { FacilityGuaranteeService } from './facility-guarantee.service';
+
+describe('FacilityGuaranteeController', () => {
+  const valueGenerator = new RandomValueGenerator();
+  const portfolioIdentifier = valueGenerator.string();
+  const facilityIdentifier = valueGenerator.facilityId();
+
+  const { facilityGuarantees: guaranteesFromService } = new GetFacilityGuaranteesGenerator(valueGenerator, new DateStringTransformations()).generate({
+    numberToGenerate: 2,
+    facilityIdentifier,
+    portfolioIdentifier,
+  });
+
+  let facilityGuaranteeServiceGetGuaranteesForFacility: jest.Mock;
```

Comment:
Can we rename this into something like `getFacilityGuaranteesService` or something like that? It's easier to read 😋